TeamCode/src/main/java/org/firstinspires/ftc/teamcode/OTOSLocalizer.java

```diff
@@ -0,0 +1,68 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.acmerobotics.dashboard.config.Config;
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.PoseVelocity2d;
+import com.acmerobotics.roadrunner.Vector2d;
+import com.qualcomm.hardware.sparkfun.SparkFunOTOS;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+@Config
+public class OTOSLocalizer implements Localizer {
+    public static class Params {
+        public double angularScalar = 0.0;
+        public double linearScalar = 0.0;
+
+        public SparkFunOTOS.Pose2D offset = new SparkFunOTOS.Pose2D(0, 0, 0);
+    }
+
+    public static Pose2d convertPose(SparkFunOTOS.Pose2D pose) {
+        return new Pose2d(pose.x, pose.y, pose.h);
+    }
+
+    public static SparkFunOTOS.Pose2D convertPose(Pose2d pose) {
+        return new SparkFunOTOS.Pose2D(pose.position.x, pose.position.y, pose.heading.toDouble());
+    }
```

Comment:
Would you mind giving these separate names? I like `fromOtosPose()`/`toOtosPose()`